### PR TITLE
Updating role to use 1.9.2

### PR DESCRIPTION
**Pull request by SiM22**
### Changes

**Improvement:** Added pipenv task as per docker build
**Improvement:** Added new packages
**Bugfix:** Restart services if gunicorn files changed

**Tests:** changed prior release to 1.8.0

### CODE REVIEW

**Review by SiM22 — COMMENTED** [review]
Updating to pass lint tests

**Comment by SiM22** [comment]
Tested as working on my own build and passing tests on CI

File: tasks/main.yml [hunk 1]
```diff
@@ -411,49 +412,50 @@
...
-      ansible.builtin.lineinfile:
-        path: "{{ paperlessng_directory }}/requirements.txt"
-        regexp: '^pikepdf(.*)'
-        line: 'pikepdf==5.4.0'
-      when: not paperlessng_version == "v1.8.0" or
-            paperlessng_version == "latest"
-
-    - name: install paperlessng requirements
+    - name: Create requirments.txt file
+      ansible.builtin.shell: ".venv/bin/pipenv requirements > requirements.txt"
```
Comment (stumpylog) [inline tasks/main.yml:438]: Maybe this should check if the requirements.txt exists already?  An install from the release archive would, but a source install wouldn't?

Comment (SiM22) [reply tasks/main.yml:438]: I'll add something in later on tonight

**Comment by stumpylog** [comment]
@SiM22 any updates on this PR?

**Comment by SiM22** [comment]
> @SiM22 any updates on this PR?

none yet, apologies. My son was born in September so i have not had any time these last few months. Will try and get a look into this

**Comment by Tronde** [comment]
Hello @SiM22 and congratulations on being a father!  
In case you'll find the time to look into this, please be aware that it doesn't fix #9 as mentioned there.

**Pull request closed without merging**